templates/bashrc.puppet.epp

```diff
@@ -40,19 +40,10 @@ function git_color ()
     fi
 }
 
-# Call the above functions inside the PS1 declaration
+# Call the above functions inside the PS1 declaration if we want to see git status
+<% if $gitprompt { -%>
 export PS1='\[$(git_color)\]$(parse_git_branch)\[\033[01;32m\]\u@\h\[\033[00m\]:\[\033[01;34m\]\w\[\033[00m\] \$ '
+<% } -%>
 
-<% if @hostname =~ /learn/ -%>
```

Comment:
This stuff is changing with the quest tool refactor. I'm working on that in a new branch. It's currently broken with regard to the other repos involved in that build, so deleting it here isn't a regression.